wrappers/python/pyproject.toml

```diff
@@ -31,8 +25,20 @@ classifiers = [
 [project.urls]
 Homepage = "https://github.com/zxing-cpp/zxing-cpp"
 
-[tool.setuptools]
-include-package-data = false
+[tool.scikit-build]
+# Protect the configuration against future changes in scikit-build-core
+minimum-version = "build-system.requires"
+
+# Setuptools-style build caching in a local directory
+build-dir = "build/{wheel_tag}"
+
+# Build stable ABI wheels for CPython 3.12+
+wheel.py-api = "cp312"
+
+[tool.cibuildwheel]
```

Comment:
How does this section conflict/interact with https://github.com/zxing-cpp/zxing-cpp/blob/95a1c4161881253301d1d2cba27336903a04fb38/.github/workflows/publish-python.yml#L51